DataHub/Integration/02_ManageAttributeSets.md

```diff
@@ -151,11 +149,12 @@ After you have created an attribute set, you can edit it.
 Depending on the selected inheritance type, only a certain number of attributes may be editable.
 However, you can add further attributes to an attribute set or remove unmapped attributes from an attribute set.
 
+> [Info] If the *Locked* toggle is enabled, this means that the attribute set is currently undergoing maintenance and therefore cannot be edited. An overview of all locked attribute sets is displayed in the *SET LOCKS* tab, see [Attribute set locks](../UserInterface/01d_AttributeSetLocks.md).
```

Comment:
Könnte mand das nicht in "Deactivate attribute set" unterbringen? Da passt es meiner Meinnung nach thematisch besser hin.

Reply:
Lock/Locked hier ist eine interne "Housekeeping"-Maßnahme, also sollte der Benutzer auch nicht machen. Locked Toggle ist hier nur informativ gedacht, damit man weiß, warum man nichts editieren kann/soll, daher ganz oben platziert.